src/Evolution/Systems/ForceFree/Fluxes.hpp

```diff
@@ -0,0 +1,86 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/ForceFree/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace ForceFree {
+
+/*!
+ * \brief Compute the fluxes of the GRFFE system with divergence cleaning.
+ *
+ * \f{align*}
+ *  F^j(\tilde{E}^i) & = -\beta^j\tilde{E}^i + \alpha (\gamma^{ij}\tilde{\psi}
+ *      - \epsilon^{ijk}_{(3)}\tilde{B}_k) \\
+ *  F^j(\tilde{B}^i) & = -\beta^j\tilde{B}^i + \alpha (\gamma^{ij}\tilde{\phi}
+ *      + \epsilon^{ijk}_{(3)}\tilde{E}_k) \\
+ *  F^j(\tilde{\phi}) & = -\beta^j \tilde{\phi} + \alpha \tilde{B}^j \\
+ *  F^j(\tilde{\psi}) & = -\beta^j \tilde{\psi} + \alpha \tilde{E}^j \\
+ *  F^j(\tilde{q}) & = \alpha \sqrt{\gamma}J^j - \beta^j \tilde{q}
+ * \f}
+ *
+ * where the conserved variables \f$\tilde{E}^i, \tilde{B}^i, \tilde{\phi},
+ * \tilde{\psi}, \tilde{q}\f$ are densitized electric field, magnetic field,
+ * magnetic divergence cleaning field, electric divergence cleaning field, and
+ * electric charge density. \f$J^i\f$ is the spatial electric current density.
+ *
+ * \f$\epsilon_{(3)}^{ijk}\f$ is the spatial Levi-Civita tensor defined as
+ *
+ * \f{align*}
+ *  \epsilon_{(3)}^{ijk} \equiv n_\mu \epsilon^{\mu ijk}
+ *   = -\frac{1}{\sqrt{-g}} n_\mu e^{\mu ijk} = \frac{1}{\sqrt{\gamma}} e^{ijk}
+ * \f}
+ *
+ * where $\epsilon^{\mu\nu\rho\sigma}$ is the Levi-Civita tensor, \f$g\f$ is the
+ * determinant of spacetime metric, \f$\gamma\f$ is the determinant of spatial
+ * metric, \f$n^\mu\f$ is the normal to spatial hypersurface. Also,
+ * \f$e^{abcd}\f$ and \f$e^{ijk}\f$ are usual antisymmetric _symbols_ (which
+ * only have the value \f$\pm 1\f$) with 4 and 3 indices, respectively, with the
+ * convention \f$e^{0123} = e^{123} = +1\f$.
+ *
```

Comment:
We use the (sensible) convention e_{0123}=+1!

Reply:
@ermost correct :)  @teukolsky sorry for the confusion! I'll update the docs.